src/applications/disability-benefits/all-claims/content/ancillaryFormsWizardSummary.jsx

```diff
@@ -113,9 +115,15 @@ const individualUnemployabilityContent = (
 );
 
 export default function summaryDescription({ formData }) {
-  const showUnemployableContent =
-    formData['view:unemployable'] &&
-    !formData['view:unemployabilityUploadChoice'];
+  // In production, users should see the unemployable content if they indicate
+  // that they're unemployable. In other environments, they should see the
+  // unemployable content only when no selection has been made for
```

Comment:
They should see the content if they claim `view:unemployable` _and_ they don't have an answer for `view:unemployabilityUploadChoice`.

Reply:
Yep, looks like I didn't push this.